arsceneview/src/main/java/io/github/sceneview/ar/ARSceneView.kt

```diff
@@ -577,14 +572,6 @@ open class ARSceneView @JvmOverloads constructor(
         super.destroy()
     }
 
-    fun destroyARCore() {
-        val scope = lifecycle?.coroutineScope ?: CoroutineScope(Dispatchers.IO)
```

Comment:
As I wrote in issue - Coroutines is not fit tool to execute background operated a specialty on Destroy callback